django_slack/api.py

```diff
@@ -132,6 +132,9 @@ def slack_message(template, context=None, attachments=None, fail_silently=None,
 
         if 'attachments' in data:
             data['attachments'] = json.dumps(data['attachments'])
+        
+        if 'blocks' in kwargs:
```

Comment:
Not sure this is right; should one be adding something to `PARAMS` earlier in the method rather than a special-case here?